cadence/mapping/grpc/domain.py

```diff
@@ -69,11 +75,14 @@ def proto_domain_status_to_dataclass(ds: domain_pb2.DomainStatus) -> DomainStatu
 def proto_domain_configuration_do_dataclass(
         domain_response: service_domain_pb2.DescribeDomainResponse) -> DomainConfiguration:
     domain_configuration = DomainConfiguration(
-        workflow_execution_retention_period_in_days=domain_response.workflow_execution_retention_period.ToSeconds() * DAY_DURATION,
+        workflow_execution_retention_period_in_days=ms_to_days(domain_response.workflow_execution_retention_period.ToMilliseconds()),
         workflow_execution_retention_period=domain_response.workflow_execution_retention_period.ToMilliseconds(),
         emit_metric=False,
         archival_bucket_name='',
```

Comment:
deprecated doesn't mean it's not being used. So we still need to assign the value correctly(same for archival_status)
```
archival_bucket_name=history_archival_uri
archival_status=history_archival_status
```

Reply:
Good catch! I didn't add it because I didn't see it on the mapper here https://github.com/uber/cadence/blob/064378890cd0186b50edd36dd679b230c44e45fd/common/types/mapper/proto/api.go#L1110 but I guess it's been deprecated there